trending_data/src/main/resources/countries_codes_original.csv

```diff
@@ -0,0 +1,257 @@
+"Country","Alpha-2 code","Alpha-3 code","Numeric code","Latitude (average)","Longitude (average)"
```

Comment:
Why is this file here twice? One of them should be removed.

Reply:
Resolved in  branch `csv-fix`